ts/packages/cli/src/commands/export.ts

```diff
@@ -41,6 +67,10 @@ export default class Export extends BaseCommand<typeof Export> {
       description: `Pipe eval commands result to export command to pass \`Configs\` as environment variables to a child-process`,
       command: `<%= config.bin %> eval ... | <%= config.bin %> <%= command.id %> --run 'node index.js'`,
     },
+    {
+      description: `Pipe eval commands result to export command and apply a prefix / suffix to each Config Key in the export result`,
+      command: `<%= config.bin %> eval ... | configu export --prefix "MYAPP_" --suffix "_PROD"`,
+    },
```

Comment:
looks like a mistake, should be a casing example and not prefix/suffix

Reply:
It is. Thanks.
Changed to `eval ... | configu export --casing "snakeCase"`